lib/puppetserver/ca/cli.rb

```diff
@@ -39,6 +42,7 @@ class Cli
 
       def self.run(cli_args = ARGV, out = STDOUT, err = STDERR)
         logger = Puppetserver::Ca::Logger.new(:info, out, err)
+        #binding.pry
```

Comment:
whoops

Reply:
I knew there would be at least a few glaringly dumb things